aamt_parse2.pm

```diff
@@ -4908,21 +5114,24 @@ sub pars_AskSelectSites
     }
 
     $array[$sIndex][MYSQL] = 0;
-    my $promptyn;    
-    ilog_print(1,"\n\n");
-    ui_printline();
-    ilog_printf(MSG_SITE_DETAILS,$strSiteName);
-    ui_printline();
-    ilog_printf(MSG_SOURCE_PATH, $documentRoot);
+#    my $promptyn;    
+#    ilog_print(1,"\n\n");
+#    ui_printline();
+#    ilog_printf(MSG_SITE_DETAILS,$strSiteName);
+#    ui_printline();
+#    ilog_printf(MSG_SOURCE_PATH, $documentRoot);
     $strYesOrNo =" ";
-    while($strYesOrNo!~/^\s*[YynN]\s*$/)
+#    while($strYesOrNo!~/^\s*[YynN]\s*$/)
     {           
-        ilog_printf(MSG_MIGRATE_SITE, $strSiteName);
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip this, if it is false, then keep the current code.